makeiso_64.sh

```diff
@@ -23,7 +23,7 @@ then
 	cd ..
 fi
 
-MAP_FILE="zig-cache/kernel.map"
+MAP_FILE="zig-out/kernel.map"
```

Comment:
I get a zig-cache directory when building unlike the CI, which gets zig-out. It might be a good idea to pass `b.install_path` as an argument to the makeiso files so they can align to whatever path is being used.

Reply:
Sure